### DISCUSSION TITLE
Request: Hide from system tray

### DISCUSSION THREAD

**Discussion by ghost** [QL-Win/QuickLook#1204] [Ideas]
I absolutely love QuickLook. I feel like it should have been the standard media player with how good it really is, but to further enhance the experience it would be nice to have the ability to hide the icon from the system tray. QuickLook is set as the default application for images and video, therefore I feel it is not necessary to display it is open.

Apologies if I'm missing a pre-existing setting that does this, and thank you for reading :-)

**Comment by rabelux** [comment]
There is an advanced configuration for this:
https://github.com/QL-Win/QuickLook/wiki/Advanced-configurations#showhide-the-tray-icon-since-v3610

**Reply by ghost** [reply]
Omg thanks so much! I feel like an idiot haha
